frontends/precinct-scanner/src/app.test.tsx

```diff
@@ -1459,3 +1156,66 @@ test('replace ballot bag flow', async () => {
   await advanceTimersAndPromises(1);
   await screen.findByText('Ballot Bag Full');
 });
+
+test('uses storage for frontend state', async () => {
+  await storage.set(stateStorageKey, {
+    isPollsOpen: true,
+    isSoundMuted: true,
+    ballotCountWhenBallotBagLastReplaced: BALLOT_BAG_CAPACITY,
+  });
+  fetchMock.get('/precinct-scanner/scanner/status', {
+    body: scannerStatus({
+      state: 'no_paper',
+      ballotsCounted: BALLOT_BAG_CAPACITY,
+    }),
+  });
+  render(<App card={card} hardware={hardware} storage={storage} />);
+  await advanceTimersAndPromises(1);
+
+  // Confirm polls status and ballot bag status loaded from storage
+  await screen.findByText('Insert Your Ballot Below');
+
+  // Confirm muted status loaded from storage
+  card.insertCard(electionManagerCard, electionSampleDefinition.electionData);
+  await authenticateElectionManagerCard();
+  await screen.findByText('Unmute Sounds');
+
+  // Confirm muted status is saved to storage
+  userEvent.click(screen.getByText('Unmute Sounds'));
+  await advanceTimersAndPromises(1);
+  expect(await storage.get(stateStorageKey)).toMatchObject({
+    isSoundMuted: false,
+  });
+  card.removeCard();
+
+  // Confirm ballot bag status is saved to storage
+  fetchMock.get(
+    '/precinct-scanner/scanner/status',
+    {
+      body: scannerStatus({
+        state: 'no_paper',
+        ballotsCounted: BALLOT_BAG_CAPACITY * 2,
```

Comment:
Are you trying to hit the next threshold by doubling the number of ballots here? Or are you just trying to surpass the current threshold? If just trying to surpass, I think `BALLOT_BAG_CAPACITY + 1` would make that clearer

Reply:
Here I'm trying to hit the next threshold.

Reply:
Ah ok I get it now. Had to read the test a bit closer